modules/030-cloud-provider-yandex/hooks/exporter_api_key.go

```diff
@@ -0,0 +1,255 @@
+/*
```

Comment:
Oh, this hook is so complicated. Could we order a token using terraform or something?

Reply:
With @konstantin-axenov we decided use terraform for creation service account and apiKey if passed apiKey is "Auto" string. Another not empty string is valid apiKey. Apikey can pass through parameter in withNatInstance layout